store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreGetTraitsTest.java

```diff
@@ -70,8 +72,10 @@
             StoreTrait.POST_AGGREGATION_FILTERING,
             StoreTrait.MATCHED_VERTEX);
     private static final Set MAP_TRAITS_EXCLUSIVE_OF_ACCUMULO = Collections.emptySet();
-    private static final FederatedAccess ACCESS_UNUSED_AUTH_AND_USER = new FederatedAccess(Sets.newHashSet(UNUSED_AUTH_STRING), UNUSED_AUTH_STRING);
-    private static final FederatedAccess ACCESS_UNUSED_AUTH_WITH_TEST_USER = new FederatedAccess(Sets.newHashSet(UNUSED_AUTH_STRING), TEST_USER_ID);
+    private static final FederatedAccess ACCESS_UNUSED_AUTH_AND_USER = new FederatedAccess(
```

Comment:
👎 line wrapping here makes it a smidge harder to spot the difference between the different FederatedAccess. 
We have bigger monitors these days.

```
xxx x xx xx xxxxx xxxx xxxxxxxx UNUSED_AUTH_STRING xxxxxx
xxx x xx xx xxxxx xxxx xxxxxxxx TEST_USER_ID xxxxxx
```